pipeline-runner/R/qc-1-filter_emptydrops.R

```diff
@@ -16,59 +16,63 @@
 #' @export
 #' @return a list with the filtered seurat object by mitochondrial content, the config and the plot values
 #'
-filter_emptydrops <- function(scdata, config, sample_id, cells_id, task_name = "classifier", num_cells_to_downsample = 6000) {
-  cells_id.sample <- cells_id[[sample_id]]
+filter_emptydrops <- function(scdata_list, config, sample_id, cells_id, task_name = "classifier", num_cells_to_downsample = 6000) {
```

Comment:
I don't see the values of `task_name` and `num_cells_to_downsample` being changed anywhere in the code, is this true? If so, I suggest we remove them from being function parameters and we make them constants.

Reply:
the task name changes, it is used in the init file, to assign to each task a function (and then call the general run_processing_step function, which dispatches to the correct task function). I'm not really sure why it's added to all steps as a (default) parameter as well.

I agree on the number of cells to downsample. It is used in several filters I think. At least 1,2 and 4

Reply:
should be improved in another PR this one is too big already